integration-tests/src/test/java/com/redhat/service/bridge/integration/tests/steps/End2EndTestITStep.java

```diff
@@ -0,0 +1,257 @@
+import java.util.List;
+
+import org.hamcrest.Matchers;
+import org.testcontainers.shaded.org.awaitility.Awaitility;
+
+import com.redhat.service.bridge.actions.kafkatopic.KafkaTopicAction;
+import com.redhat.service.bridge.infra.api.APIConstants;
+import com.redhat.service.bridge.infra.models.actions.BaseAction;
+import com.redhat.service.bridge.infra.models.dto.BridgeStatus;
+import com.redhat.service.bridge.manager.api.models.responses.BridgeListResponse;
+import com.redhat.service.bridge.manager.api.models.responses.BridgeResponse;
+import com.redhat.service.bridge.manager.api.models.responses.ProcessorResponse;
+
+import io.cloudevents.SpecVersion;
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import io.restassured.http.ContentType;
+import io.restassured.http.Header;
+import io.restassured.http.Headers;
+
+import static com.redhat.service.bridge.integration.tests.common.BridgeCommon.addBridge;
+import static com.redhat.service.bridge.integration.tests.common.BridgeCommon.createProcessor;
+import static com.redhat.service.bridge.integration.tests.common.BridgeCommon.deleteBridge;
+import static com.redhat.service.bridge.integration.tests.common.BridgeCommon.deleteProcessor;
+import static com.redhat.service.bridge.integration.tests.common.BridgeCommon.getBridgeDetails;
+import static com.redhat.service.bridge.integration.tests.common.BridgeCommon.getBridgeList;
+import static com.redhat.service.bridge.integration.tests.common.BridgeCommon.getProcessor;
+import static com.redhat.service.bridge.integration.tests.common.BridgeCommon.managerUrl;
+import static com.redhat.service.bridge.integration.tests.common.BridgeUtils.jsonRequestWithAuth;
+import static io.restassured.RestAssured.given;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class End2EndTestITStep {
+
+    private static final String TOPIC_NAME = "myKafkaTopic";
+    private static final String PROCESSOR_NAME = "myProcessor";
+
+    private static String bridgeId;
+    private static String processorId;
+    private static String endPoint;
+    private static String ingressMetrics;
+    private static InputStream cloudEventStream;
+
+    @Given("get list of Bridge instances returns HTTP response code 401")
+    public void authenticationIsEnabled() {
+        given()
+                .contentType(ContentType.JSON)
+                .when()
+                .get(managerUrl + APIConstants.USER_API_BASE_PATH)
+                .then()
+                .statusCode(401);
+    }
+
+    @Given("^get list of Bridge instances with access token doesn't contain Bridge \"([^\"]*)\"$")
+    public void getEmptyBridges(String bridgeName) {
+        BridgeListResponse response = getBridgeList();
+        assertThat(response.getKind()).isEqualTo("BridgeList");
+        if (response.getItems().size() > 0) {
+            for (int i = 0; i < response.getItems().size(); i++) {
```

Comment:
Optional - use can use "for each" here.

Reply:
Or better to use assertion for whole expression:
`assertThat(response.getItems()).noneMatch(b -> b.getName().equals(bridgeName));`